vm/src/builtins/pytype.rs

```diff
@@ -480,8 +480,10 @@ impl PyType {
         }
 
         // TODO: how do we know if it should have a dict?
-        let flags = base.slots.flags | PyTpFlags::HAS_DICT;
-
+        // All *classes* should have a dict. Exceptions are *instances* of
```

Comment:
revised this. `HAS_DICT` can be used (temporarily?) as `tp_dictoffset` is used in CPython.